docs/literate/src/files/first_steps/create_first_setup.jl

```diff
@@ -0,0 +1,245 @@
+#src # Create first setup
+
+# In this part of the introductory guide, we will create a first Trixi.jl setup as an extension of
+# [`elixir_advection_basic.jl`](https://github.com/trixi-framework/Trixi.jl/blob/main/examples/tree_2d_dgsem/elixir_advection_basic.jl).
+# Since Trixi.jl has a common basic structure for the setups, you can create your own by extending
+# and modifying the following example.
+
+# Let's consider the linear advection equation for a state ``u = u(t, x, y)`` on the two-dimensional spatial domain
```

Comment:
```suggestion
# Let's consider the linear advection equation for a state ``u = u(x, y, t)`` on the two-dimensional spatial domain
```
In the initial condition function further down, we use the spatial coordinates before the time variable. Being consistent between the text and the implementation in terms of the order of arguments might improve readability.